apps/sms/test/marionette/lib/messages.js

```diff
@@ -30,12 +30,12 @@ var ConversationAccessor = require('./views/conversation/accessors');
     },
 
     Report: {
-      main: '#information-report',
+      main: '.panel-ReportView',
```

Comment:
__note:__ mixing ```panel``` and ```view``` in one name looks a bit strange, but I don't really care :)